arm/Microsoft.Network/connections/deploy.bicep

```diff
@@ -1,7 +1,7 @@
 @description('Required. Remote connection name')
 param connectionName string
 
-@description('Required. Specifies a VPN shared key. The same value has to be specified on both Virtual Network Gateways')
+@description('Optional. Specifies a VPN shared key. The same value has to be specified on both Virtual Network Gateways')
 param vpnSharedKey string
```

Comment:
Pretty sure bicep will still ask you for the parameter if there is no default value provided. In this case it may probably make sense to set an empty string `param vpnSharedKey string = ''`

Reply:
@jceval, one more: Note that the readme references the parameter description too. So the 'Required' has to be updated to 'Optional' there as well.

Reply:
Got it. Thanks for the help !